app/src/main/res/values/colors.xml

```diff
@@ -60,6 +60,7 @@
     <color name="readermode_end_gradient_normal_theme">#FF0250bb</color>
     <color name="search_suggestion_indicator_icon_color_light_theme">#008787</color>
     <color name="search_suggestion_indicator_icon_bookmark_color_light_theme">#0060df</color>
+    <color name="preference_section_header_normal_theme">#592acb</color>
```

Comment:
We should have @color variables for these. I think #592ACB maps out to @color/photonViolet70, but also best to do a search.